desktop/release-notes.md

```diff
@@ -23,13 +23,98 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+- Add new flag for Windows installer, `--no-windows-containers` to disable Windows containers integration.
+
+### Upgrades
+- [Docker Compose v2.7.0](https://github.com/docker/compose/releases/tag/v2.7.0)
+- [Docker Compose "Cloud Integrations" v1.0.28](https://github.com/docker/compose-cli/releases/tag/v1.0.28)
+- [Kubernetes v1.24.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.2)
+- [Go 1.18.4](https://github.com/golang/go/releases/tag/go1.18.4)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Adds the Container / Compose icon as well as the exposed port(s) / exit code to the Containers screen.
+- Updated Docker theme palette colours values to match design system.
+- Improved an error message from `docker login` if Registry Access Management is blocking the Docker engine's access to Docker Hub.
+- Increase throughput between the Host and Docker, for example increasing performance of `docker cp`.
+- Collecting diagnostics takes less time to complete.
+- Selecting/unselecting a compose app on the containers overview will select/unselect all its containers.
+- Show tag names on the container overview image column.
+- Added search decorations to the terminal's scrollbar so that matches outside the viewport are visible.
+- Fixed issue on search which doesn't work well on containers page [docker/for-win#12828](https://github.com/docker/for-win/issues/12828).
+- Fixed issue on infinite loading on volume screen [docker/for-win#12789](https://github.com/docker/for-win/issues/12789).
+- Fixed a problem in the Container UI where resizing or hiding columns didn't work. Fixes [docker/for-mac#6391](https://github.com/docker/for-mac/issues/6391).
+- Fixed a bug where the state of installing, updating, or uninstalling multiple extensions at once was lost when leaving the Marketplace screen.
+- Fixed issue where compose version in the about page would only get updated from v2 to v1 after restart.
+- Fixed an issue where user cannot not see log view because their underlying hardware didn't support WebGL2 rendering. Fixes [docker/for-win#12825](https://github.com/docker/for-win/issues/12825).
+- Fixed a bug where the UI for Containers and Images got out of sync.
+- Fix a startup race when the experimental virtualization framework is enabled.
+
+#### For Mac
+
+- Fixed an issue executing Compose commands from the UI. Fixes [docker/for-mac#6400](https://github.com/docker/for-mac/issues/6400).
+
+#### For Windows
+
+- Fixed horizontal resizing issue. Fixes [docker/for-win#12816](https://github.com/docker/for-win/issues/12816).
+- If an HTTP/HTTPS proxy is configured in the UI, then automatically send traffic from image builds and running containers to the proxy, avoiding the need to separately configure environment variables in each container or build.
+- Fixed an issue doing actions in the UI with Compose apps that were created from a WSL distro. Fixes [docker/for-win#12806](https://github.com/docker/for-win/issues/12806).
```

Comment:
Can this sentence be clarified further? I'm not sure what this is meant to mean.

Reply:
@joe0BAB implemented this change, and would probably be the best person to clarify this note.

Reply:
But the fix already got shipped with the patch release 4.10.1, not sure it should be on this list again

Reply:
Ah ok, my bad I didn't spot it on the 4.10.1 list. I'll remove it from this one.